LedgerHQ/ledger-live-desktop
# Restore cryptographic verification of updates on Mac

`electron-updater` has been rollbacked in #2008 to an old upstream version (because of a bug in our fork of the lib) which doesn't check Ledger's additional cryptographic signature of updates on Mac.

The forked lib bug has been fixed in ledgerhq/electron-builder#2.
This PR switch to that verison, restoring the additional crypto checks minus the regression.

### Type

Bugfix

### Context

https://ledgerhq.atlassian.net/browse/LL-1423

### Parts of the app affected

Auto update

### Test plan

- Decrement app version number
- Build with `yarn dist`
- It should update successfully to last production release and auto-restart on
- Using `UPDATE_CHECK_FEED` constant, point to an invalid signature file, it should
  - Update successfully anyway
  - Send an error to Sentry
- Using `UPDATE_CHECK_FEED` constant, point to an invalid URL, it should
  - Update successfully anyway
  - Send an error to Sentry
